dojo/settings/settings.dist.py

```diff
@@ -373,6 +370,13 @@ def generate_url(scheme, double_slashes, user, password, host, port, path):
     'ATTRIBUTES_MAP': env('DD_SAML2_ATTRIBUTES_MAP'),
 }
 
+# Metadata is required, choose either remote url or local file path
+if 'DD_SAML2_METADATA_AUTO_CONF_URL' in os.environ:
```

Comment:
Change to the following so this isn't breaking for users that rely on the environ env vars within settings.py.

`if 'DD_SAML2_METADATA_AUTO_CONF_URL' in os.environ or len(env('DD_SAML2_METADATA_AUTO_CONF_URL')) > 0:`

Tested on my instance.

Reply:
Done